spec/support/shared/examples/msf/db_manager/exploit_attempt.rb

```diff
@@ -31,14 +35,21 @@
       }
     end
 
-    subject(:report_exploit_success) do
-      db_manager.report_exploit_success(opts)
+    context 'with a vuln' do
+      specify do
+        expect {
+          report_exploit_success
+        }.to change(Mdm::VulnAttempt,:count).by(1)
+      end
     end
 
-    specify do
-      expect {
-        report_exploit_success
-      }.to change(Mdm::VulnAttempt, :count).by(1)
+    context 'without a vuln' do
+      let(:vuln) { nil }
+      specify do
```

Comment:
Please add a blank line between the `let` and `specify` so the `let` pops out more when reading.